x/nft/bootstrap_node/model.go

```diff
@@ -70,7 +72,16 @@ func (m *BootstrapNodeToken) Validate() error {
 }
 
 func (m URI) Validate() error {
-	//todo: impl
+	if m.Port == 0 {
+		return nft.ErrInvalidPort()
+	}
+	if err := validation.Validate(m.Host, is.Host); err != nil {
+		return nft.ErrInvalidHost()
+	}
+	if err := validation.In("http", "https", "ws", "wss", "grpc"); err != nil {
```

Comment:
What is this comparing against? Like which string must be in that list of strings? This seems a bit too magic

Reply:
it was my mistake, was already fixed